lib/cli/index.js

```diff
@@ -2536,6 +2564,98 @@ export async function createJavaBom(path, options) {
       force: true,
     });
   }
+
+  if (
+    millFiles?.length &&
+    isPackageManagerAllowed(
+      "mill",
+      ["bazel", "sbt", "gradle", "maven"],
+      options,
+    )
+  ) {
+    const millRootPath = dirname(millFiles[0]);
```

Comment:
Should we find this root path based on the first directory containing the .mill file?

Reply:
I copied that from gradle and assumed the search would return the 'highest' file in the tree as the first one. But we can add some more checks for that if you want -- we should than obviously do that in other parts of the codebase as well

Reply:
Good point. Didn't realise this limitation in gradle. In that case we can do it in another PR as well.